src/frontend/apps/standalone_site/src/features/Menu/components/Burger/Burger.tsx

```diff
@@ -0,0 +1,22 @@
+import { SVGProps, useState } from 'react';
+
+import { ReactComponent as BurgerIcon } from 'assets/svg/burger.svg';
+
+import styles from './Burger.module.css';
+
+type BurgerProps = SVGProps<SVGSVGElement>;
+
+function Burger(props: BurgerProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <BurgerIcon
+      role="button"
+      className={`${styles.burgerIcon} ${isOpen ? styles.open : ''}`}
```

Comment:
We discuss about this and we think that using the [import class module](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/) feature from CRA is a great think in case we are reusing css from other project.